tools/coav-control/coav-control.sh.in

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+### BEGIN INIT INFO
+# Provides:        Collision Avoidance Daemon
+# Required-Start:  $network #FIXME
+# Required-Stop:   $network #FIXME
+# Default-Start:   2 3 4 5
+# Default-Stop:
+# Description:     Enable collision avoidance capabilities.
+### END INIT INFO
+
+. /etc/init.d/functions
+
+get_pid() {
+    pidof "coav-control"
+}
+
+start() {
+    nohup @CMAKE_INSTALL_FULL_BINDIR@/coav-control \
+        -d DI_OBSTACLE \
+        -a QC_STOP \
+        -s ST_REALSENSE \
```

Comment:
Yep, only if we add support to config file-s at coav-control, but I don't think we want to go that far. This init script its already more that I was willing to provide for it since invite people to use this tool as the a final coav app.